Solutions/Armorblox/Playbooks/README.md

```diff
@@ -0,0 +1,28 @@
+# Needs-Review-Incident-Email-Notification
+**Author:** Armorblox
+
+This playbook will send an Email notification when a new incident is opened in Azure Sentinel.
+## Pre-requisites
+An O365 account to be used to send email notification (The user account will be used in O365 connector (Send an email).)
```

Comment:
What is the `(Send an email)` bit?

Reply:
It represents the step 2 in the playbook. o365 is the brand and the feature used from the brand is `send an email`

Reply:
Should I remove it `[(Send an email)]`

Reply:
That's fine. I think you can change this line to `An O365 account to be used to send email notification. The user account will be used in O365 connector (Send an email).`